android/src/main/java/com/swmansion/rnscreens/ScreenStackFragment.kt

```diff
@@ -62,7 +62,8 @@ class ScreenStackFragment : ScreenFragment, ScreenStackFragmentWrapper {
 
     override fun setToolbarShadowHidden(hidden: Boolean) {
         if (isToolbarShadowHidden != hidden) {
-            appBarLayout?.targetElevation = if (hidden) 0f else PixelUtil.toPixelFromDIP(4f)
+            appBarLayout?.elevation = if (hidden) 0f else PixelUtil.toPixelFromDIP(4f)
+            appBarLayout?.stateListAnimator = null
```

Comment:
Why `null` here, instead of leaving default?

Reply:
Without it the elevation is not setting properly on initial render, I suppose the shadow is there by default so we have to reset it.

Reply:
Ok